lisa/sut_orchestrator/hyperv/context.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass
+from pathlib import PureWindowsPath
+from typing import Optional
+
+from lisa import Node, RemoteNode
+from lisa.util.parallel import TaskManager
+
+
+@dataclass
+class NodeContext:
+    vm_name: str = ""
+    host: Optional[RemoteNode] = None
+    working_dir = PureWindowsPath()
+    vhd_path = PureWindowsPath()
```

Comment:
Why does it need to save vhd_path?

Reply:
You're right we are not using the stored value later. Removed it.